client/src/pages/Create.js

```diff
@@ -138,27 +173,42 @@ export function Create() {
             <input
               type="text"
               placeholder="Straße und Hausnr."
-              name="Straße"
-              ref={register}
+              name="street"
+              ref={register({
+                required: true,
+              })}
+            />
+            <input
+              type="text"
+              placeholder="PLZ"
+              name="zip"
+              ref={register({
+                required: true,
+              })}
             />
-            <input type="text" placeholder="PLZ" name="PLZ" ref={register} />
             <input
               type="text"
               placeholder="Stadt"
-              name="Stadt"
-              ref={register}
+              name="city"
+              ref={register({
+                required: true,
+              })}
             />
             <input
               type="text"
               placeholder="Klingelname / Treffpunkt"
-              name="Klingelname / Treffpunkt"
-              ref={register}
+              name="name"
+              ref={register({
+                required: true,
+              })}
             />
           </Location>
           <Footer>
             <FooterGradient />
             <Button type="submit">Anbieten</Button>
-            <CancelButton type="reset">Abbrechen</CancelButton>
+            <CancelButton onClick={() => history.goBack()}>
```

Comment:
You can optimize this a bit

```suggestion
            <CancelButton onClick={history.goBack}>
```